python/opentrons_ot3_firmware/messages/payloads.py

```diff
@@ -203,3 +203,10 @@ class FirmwareUpdateCompleteAcknowledge(utils.BinarySerializable):
     """A response to the FirmwareUpdateComplete message."""
 
     error_code: utils.UInt16Field
+
+
+@dataclass
+class GetLimitSwitchResponse(utils.BinarySerializable):
+    """A response to the Limit Swith Status request payload"""
+
```

Comment:
Do we have a definition of the switch status?
I am assuming an Enum that maps to the int?